### PR TITLE
Add proper message for empty defi overview

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/locales/en.json [hunk 1]
```diff
@@ -3100,7 +3100,8 @@
   },
   "decentralized_overview": {
     "loading": "Please wait while your defi balances are getting loaded...",
-    "title": "Defi Overview"
+    "title": "Defi Overview",
+    "empty_title": "No DeFi overview can be shown"
```
Comment (LefterisJP) [inline frontend/app/src/locales/en.json:3104]: When can that happen @lukicenturi ?

**Review by LefterisJP — COMMENTED** [review]
If possible some times add screenshots too.

File: frontend/app/src/locales/en.json (hunk 1 above)
Comment (LefterisJP) [reply frontend/app/src/locales/en.json:3104]: Ah I see. Can you please add a tiny bit more info to the message so the users knows why?

"No DeFi overview can be shown, please add at least one ethereum account" ?

Comment (lukicenturi) [reply frontend/app/src/locales/en.json]: > Ah I see. Can you please add a tiny bit more info to the message so the users knows why?
> 
> "No DeFi overview can be shown, please add at least one ethereum account" ?

Updated 
image

**Review by LefterisJP — APPROVED** [review]
Perfect @lukicenturi, thank you :+1:

**Pull request merged**